src/Api/Auth/Controllers/TwoFactorController.cs

```diff
@@ -236,7 +236,7 @@ public async Task<TwoFactorDuoResponseModel> PutDuo([FromBody] UpdateTwoFactorDu
     [HttpPost("get-webauthn")]
     public async Task<TwoFactorWebAuthnResponseModel> GetWebAuthn([FromBody] SecretVerificationRequestModel model)
     {
-        var user = await CheckAsync(model, true);
+        var user = await CheckAsync(model, false);
```

Comment:
Minor (non-blocking):  @justindbaur has made the excellent point to me several times of including the parameter name in the method call like `await CheckAsync(model, premium: false);` because it is easier to read what is happening on Github or when not in an IDE which shows what the parameter is like Rider:
![image](https://github.com/bitwarden/server/assets/116684653/735bf77f-b19a-4981-845b-6e82d3320f88)